proposals/description/fip_76a.ts

```diff
@@ -0,0 +1,41 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_x: ProposalDescription = {
+  title: 'FIP-76a: Add LUSD to FeiRari',
+  commands: [
+    {
+      target: 'fuseAdmin',
+      values: '0',
+      method: 'oracleAdd(address[],address[])',
+      arguments: [['{lusd}'], ['{rariChainlinkPriceOracleV3}']],
+      description: 'Set oracle for LUSD'
+    },
+    {
+      target: 'fuseAdmin',
+      values: '0',
+      method: '_deployMarket(address,address,string,string,address,bytes,uint256,uint256,uint256)',
+      arguments: [
+        '{lusd}', // underlying
+        '{rariPool8DaiIrm}', // IRM
+        'FeiRari LUSD', // Name
+        'fLUSD-8', // Symbol
+        '{rariPool8CTokenImpl}', // impl
+        '0x', // constructor bytes (not used)
+        '200000000000000000', // reserve factor 20%
```

Comment:
0 is standard for FeiRari. We already have a 6% admin fee and don't need to build reserves

Reply:
DAI and ETH have 20% reserve factor